tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -30,47 +30,54 @@ DEBUG_START
 # PCM6 ---> volume <---- iDisp2
```

Comment:
@tlauda can you explain again in the commit message why 3 periods are needed. Maybe we should also think about this for our i.MX topologies.

Reply:
@tlauda Anything specific? Because I failed to get hardware SG to work on my EDMA driver too. Could it well be from this possibility?

This is actually important to me since currently the EDMA driver only supports 2 contiguous elems in non-SG mode so the topologies on i.MX *cannot* be changed to 3 periods.

Reply:
@paulstelian97 It's only required if you are scheduling pipelines on timer_domain or dma_single_chan_domain, because they are both using one irq to schedule all the pipes. For dma_multi_chan_domain it should work as usual, that means pipeline copy per own irq.

Reply:
Ah OK, thanks.